fava/static/javascript/editor.js

```diff
@@ -179,6 +212,8 @@ $(document).ready(function() {
             event.preventDefault();
             event.stopImmediatePropagation();
 
+            CustomEvents.fire(document, "LiveReloadShutDown");
```

Comment:
That's what I tried first, but it didn't work for me. (Chromium 48.0.2564.82 Ubuntu 14.04 (64-bit))

I'll do a reset and reinstall of my local version to make sure it's not conflicting with some local changes and let you know later.